crates/flipperzero/src/furi/thread.rs

```diff
@@ -203,6 +205,15 @@ pub fn sleep(duration: core::time::Duration) {
     }
 }
 
+/// Puts the current thread to sleep for at least the specified amount of time.
+///
+/// This differs from [`sleep`] in that a Furi [`Duration`] can represent an exact number of ticks.
+pub fn sleep_ticks(duration: Duration) {
```

Comment:
Are we more interested in users using this instead of `sleep`? If yes then maybe we want it to have an easier `sleep` name with `sleep_<smth>` be given to the former.

Also, it would be useful to also give a reference to this function from the former's doc.

Reply:
I'm a bit torn on this one. Originally the plan was to make `sleep` take a `furi::time::Duration`, but `core::time::Duration` can support sub-tick (typically 1 ms) durations which can be used with `furi_delay_ms`.

Updated the documentation to explain the difference between the two.